demo-shell/src/app/components/task-list-demo/task-list-demo.component.html

```diff
@@ -0,0 +1,72 @@
+<div class="task-list-demo-inputs">
+
+     <form>
+        <mat-form-field>
+            <input 
+            matInput 
+            placeholder="Task Name" 
+            [formControl]="taskName" 
+            (keyup)="filterTasks()">
+        </mat-form-field>
+
+        <mat-form-field>
+            <input 
+            matInput 
+            placeholder="ProcessDefinitionId" 
+            matTooltip="E.g. SimpleProcess:1:2"
+            [formControl]="taskProcessDefinitionId" 
+            (keyup)="filterTasks()">
```

Comment:
can we do something like that
this.taskProcessDefinitionId.valueChanges
            .pipe(debounceTime(300))